addon/components/tether-tooltip-and-popover.js

```diff
@@ -454,10 +454,13 @@ export default EmberTetherComponent.extend({
 
   stopTether() {
     run.schedule('afterRender', () => {
-      // can't depend on `_tether.enabled` because it's not an
-      // Ember property (so won't trigger cp update when changed)
-      this.set('_isTetherEnabled', false);
-      this.get('_tether').disable();
+      if (!this.isDestroyed) {
```

Comment:
Happy to include `isDestroying` as well. Just because I've not seen that error state doesn't mean we should be too sure it couldn't happen.

I've updated the check to be like this example from Ember itself:

https://github.com/emberjs/ember.js/blob/d5acb2d4ec989aa33959cd31cddcd1c854f9919f/packages/ember-runtime/lib/mixins/promise_proxy.js#L22

This and other similar examples are what I'm following when I directly access the property rather than through `get`.